platform/ios/ios.xcworkspace/xcshareddata/xcdebugger/Breakpoints_v2.xcbkptlist

```diff
@@ -11,14 +11,6 @@
             continueAfterRunningActions = "No"
             scope = "1"
             stopOnStyle = "0">
-            <Actions>
```

Comment:
I feel like early on this was something I reversed in `master` during this process. I'll take it out of this PR; didn't mean to change anything currently.